bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/PanelImpl.java

```diff
@@ -72,4 +73,13 @@ public Boolean isReadOnly() {
         return readOnly;
     }
 
+    @Override
+    @JsonIgnore
+    public String getTitle() {
```

Comment:
Why do you need this ? Please re-use getLabel itself

Reply:
I need panel title in datalayer. but don't want to add the same in json model.
I added the same instead of using getLabel to add dc:title to datalayer.

Do you mean I should call getLabel for dc:title instead of introducing new function?

Reply:
We already have getTitle in FormConatiner, not adding getTitle to other component will make me check type accordingly and call "getTitle" for formContainer and "getlabel" for other component